cmd/snap/cmd_connections.go

```diff
@@ -114,6 +115,22 @@ func (b byConnectionData) Less(i, j int) bool {
 	return iCon.slot < jCon.slot
 }
 
+func definingAttributeValue(conn *client.Connection) string {
+	var value string
+
+	switch conn.Interface {
+	case "content":
+		value, _ = conn.PlugAttrs["content"].(string)
+		if value == "" {
+			value, _ = conn.SlotAttrs["content"].(string)
+		}
+	}
+	if value != "" {
```

Comment:
I'd've turned the `if` around,
```go
if value == "" {
        return ""
}
return fmt.Sprintf("[%v]", value)
```
mostly because I don't like the `return value` being equivalent to `return ""`